app/facads/tmdb_search.rb

```diff
@@ -0,0 +1,11 @@
+class TmdbSearch
+
+  def service
+    TmdbService.new
+  end
+
+  def keyword(name)
+    @_search = JSON.parse(service.get_url.get("/fdc/v1/foods/search?query#{name}").body, symbolize_names: true)
```

Comment:
Does the facade need to know the API search path or can all the API logic live in the service?